app/master/build.py

```diff
@@ -471,19 +473,21 @@ def _perform_async_postbuild_tasks(self):
         data. This method also transitions the FSM to finished after the postbuild tasks are complete.
         """
         try:
-            self._create_build_artifact()
+            timing_data = self._read_subjob_timings_from_results()
+            self._create_build_artifact(timing_data)
```

Comment:
Yeah. I originally took it out because I thought timing_data had both `atom.actual_time` and `atom.expected_time`. I was gonna use timing_data to calculate the build_estimation_error metric, but
1.  timing_data doesn't have actual_time
2. I'm getting rid of build_estimation_error anyway